crates/cli/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod app_builder;
```

Comment:
cli is also used as a lib?

Reply:
cli is only used by this repo. The wasm playground is connected straight to zkwasm lib.

Reply:
So why add the lib.rs file

Reply:
correct my last comment. It is also used as a library by some customers.